src/Uno.Toolkit.UI/Behaviors/DrawerControlBehavior.cs

```diff
@@ -33,73 +34,87 @@ public static class DrawerControlBehavior
 	{
 		#region DependencyProperty: DrawerBackground
 
-		public static DependencyProperty DrawerBackgroundProperty { get; } = DependencyProperty.RegisterAttached(
+		public static DependencyProperty DrawerBackgroundProperty { [DynamicDependency(nameof(GetDrawerBackground))] get; } = DependencyProperty.RegisterAttached(
 			"DrawerBackground",
 			typeof(Brush),
 			typeof(DrawerControlBehavior),
 			new PropertyMetadata(default(Brush)));
 
+		[DynamicDependency(nameof(SetDrawerBackground))]
 		public static Brush GetDrawerBackground(DrawerControl obj) => (Brush)obj.GetValue(DrawerBackgroundProperty);
+
+		[DynamicDependency(nameof(GetDrawerBackground))]
 		public static void SetDrawerBackground(DrawerControl obj, Brush value) => obj.SetValue(DrawerBackgroundProperty, value);
 
 		#endregion
 		#region DependencyProperty: OpenDirection
 
-		public static DependencyProperty OpenDirectionProperty { get; } = DependencyProperty.RegisterAttached(
+		public static DependencyProperty OpenDirectionProperty { [DynamicDependency(nameof(GetOpenDirection))] get; } = DependencyProperty.RegisterAttached(
 			"OpenDirection",
 			typeof(DrawerOpenDirection),
 			typeof(DrawerControlBehavior),
 			new PropertyMetadata(default(DrawerOpenDirection)));
 
+		[DynamicDependency(nameof(SetOpenDirection))]
 		public static DrawerOpenDirection GetOpenDirection(DependencyObject obj) => (DrawerOpenDirection)obj.GetValue(OpenDirectionProperty);
```

Comment:
Are the Sets and Gets supposed to be flipped? 

Like DynamicDependency("SetOpenDirection") is on the GetOpenDirection method.

Reply:
Oh I see, just looked up DynamicDependency. Nevermind

Reply:
They are! The idea is that if you use the Set, the Get should stay (even if not referenced directly); if you use the Get, the set should stay; and if you use the DP auto property, both the get and set should say (transitively). We'll modify the uno generator (and probably extract it), so it does this automatically somehow.